frontend/src/screens/wallet/send/ConfirmPayment.tsx

```diff
@@ -83,7 +83,7 @@ export default function ConfirmPayment() {
           <FormattedFiatAmount amount={amount || invoice.satoshi} />
         </div>
         {invoice.description && (
-          <div className="mt-2">
+          <div className="mt-2 max-w-lg break-words">
```

Comment:
Why `max-w-lg`? https://tailwindcss.com/docs/max-width

Reply:
Shouldn't that be limited by the container (instead of setting it explicitly)

Reply:
true. sendLayout already uses max-w-lg. depending on the text we have to use break-all instead of break-words to make it work without max-w-lg setting again at the <p> level. fixed